fm.reaper.Reaper.yml

```diff
@@ -0,0 +1,69 @@
+app-id: fm.reaper.Reaper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+tags:
+ - proprietary
+command: /app/REAPER/reaper
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0
+  - --device=all
+  - --filesystem=home
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+  - --system-talk-name=org.freedesktop.Flatpak
```

Comment:
do you really need this ?

Reply:
` - --device=all`
Wouldn't it be necessary to be able to recognize any Audio Interface? I also put because Reaper can edit video.
`- --filesystem=home`
As the project and plugins could be anywhere on the home, I decided to give access to it.
```
  - --system-talk-name=org.freedesktop.RealtimeKit1
  - --system-talk-name=org.freedesktop.Flatpak
```
Those really should have been removed. Only Realtime that I was confused if it would be necessary for the video editing.

Reply:
I was only questioning the last line. 

As for `org.freedesktop.RealtimeKit1` it depends.